extensions/format/README.md

```diff
@@ -0,0 +1,30 @@
+# Formatting for Lem
+This package exposes the `format-buffer` command and supports auto-formatting on save.
+
+# Register formatters
+Formatting for JavaScript, JSON, C, Go, and Common Lisp are included as defaults, but you can register other formatters with `register-formatter` and `register-formatters`
+
+```lisp
+;; register multiple formatters at once:
+(register-formatters
+  ('lem-go-mode:go-mode #'gofmt)
```

Comment:
Is quoting `'lem-go-mode:go-mode` really needed inside the macro?

Reply:
proposal/idea: in the macro, instead of `,(first binding) ` have `',(first binding) ` (quote added) so than we can write 

```lisp
(register-formatters
  (lem-go-mode:go-mode     #'gofmt)
  …)
```

which looks better to me.

I tested lightly (it compiles).

Reply:
Fixed this in `register-formatter` instead of the plural form so now quotes are not required for either.